kinetic/distribution.yaml

```diff
@@ -695,6 +695,11 @@ repositories:
       url: https://github.com/orbbec/ros_astra_launch.git
       version: master
     status: developed
+  astrobee:
+    doc:
+      type: git
+      url: https://github.com/nasa/astrobee
+      version: master
```

Comment:
Kinetic is end-of-life, so we aren't allowing any further changes to it.  Please remove this section.

Reply:
done